cogs/vxtwitconverter/constants.py

```diff
@@ -1,8 +1,8 @@
 import enum
 
 KEY_ENABLED = "enabled"
-
 DEFAULT_GUILD = {KEY_ENABLED: False}
+INSTA_REGEX_MATCH = r"https://(?:www\.)?(instagram.com)"
```

Comment:
With RegEx pattern caching in newer Python, we certainly don't need to worry about a pattern string gets compiled again and again, but, I prefer that this becomes a compiled pattern. Thoughts?
```suggestion
INSTA_REGEX_PATTERN = re.compile(r"https://(?:www\.)?(instagram.com)")
```
Naming is up to you.

Reply:
Thanks, took your suggestion.